src/main/AuthenticationRequestRegistration.ts

```diff
@@ -32,10 +32,16 @@ export function createRequestRegistration(opts: SIOP.RequestRegistrationOpts): {
   };
 }
 
+//TODO: fill it with right values
 function createRPRegistrationMetadataPayload(opts: SIOP.RPRegistrationMetadataOpts): SIOP.RPRegistrationMetadataPayload {
   return {
-    did_methods_supported: opts.didMethodsSupported || ['did:eosio:', 'did:ethr:', 'did:factom:', 'did:lto:'],
-    subject_identifiers_supported: opts.subjectIdentifiersSupported || SIOP.SubjectIdentifierType.DID,
-    credential_formats_supported: opts.credentialFormatsSupported || [],
+    authorization_endpoint: opts.authorizationEndpoint,
+    id_token_signing_alg_values_supported: opts.idTokenSigningAlgValuesSupported,
+    request_object_signing_alg_values_supported: opts.requestObjectSigningAlgValuesSupported,
+    response_types_supported: opts.responseTypesSupported,
+    scopes_supported: opts.scopesSupported,
+    subject_types_supported: opts.subjectTypesSupported,
+    subject_syntax_types_supported: opts.subjectSyntaxTypesSupported || ['did:eosio:', 'did:ethr:', 'did:factom:', 'did:lto:'],
```

Comment:
The did:eosio can be removed. And added: did:key and did:ion, so we can conform to the new JWT VC Presentation Profile

Reply:
We also need to create a Ticket/TODO to start supporting JWK thumbprints, which is easy enough to create: https://www.rfc-editor.org/rfc/rfc7638

Reply:
removed `did:eosio`, added: `did:key` and `did:ion`

Reply:
Sory I meant did:web instead of did:key. Maybe we should also just remove did:ethr. did:factom and did:lto, to make it less opiniated for others. So if not supplied we only support did-web and did-ion conforming to the Interop profile

Reply:
changed to: `subject_syntax_types_supported: opts.subjectSyntaxTypesSupported || ['did:web:', 'did:ion:']`